docs/source/quilt_getting_started.ipynb

```diff
@@ -85,18 +78,23 @@
    "source": [
     "## Get Quilt Calibrations\n",
     "\n",
-    "A production QPU has a set of calibrations associated with it. These include frame definitions, gate and measurement calibrations, and custom waveforms. Below we show how to get the default calibrations.\n",
-    "\n",
-    "**Note**: This depends on run-time configuration of the QPU, and is likely to vary over time."
+    "A production QPU has a set of calibrations associated with it. These include frame definitions, gate and measurement calibrations, and custom waveforms. Below we show how to get the default calibrations."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "cals = qc.compiler.get_quilt_calibrations()"
+    "cals = qc.compiler.calibrations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `calibrations` property of `QPUCompiler` provides *cached* access to the calibrations. Upon first using this property a request will be made for the calibration information and may take some time to complete. Subsequent usage of this property will use the cached calibrations and thus will be instantaneous. It should be noted therefore that calibrations **will vary with time** and should be **regularly refreshed** though the specifics of when to refresh the calibrations is left as an exercise for the user. See `QPUCompiler#refresh_calibrations`."
```

Comment:
Instead of exercise for the user (since that's not really an exercise and we are the only source of that information), maybe suggest every 15 minutes. The timeline on which calibrations are refreshed varies per quantum processor, implementation, and our changing internal practices, but every 15 minutes should mean that calibrations are generally never far enough out of date to make an important difference.